bin/cumulus.rb

```diff
@@ -221,6 +221,6 @@ def self.security
   Modules.iam
 elsif ARGV[0] == "autoscaling"
   Modules.autoscaling
-elsif ARGV[0] == "security"
+elsif ARGV[0] == "security-groups"
   Modules.security
```

Comment:
I think it's fine to leave the module as is, cause it's never exposed to the end user. If I changed this here, I might as well change the folder name, and then all the places where it's referenced. It'd be a huge huge pain.